src/pages/extending-agent-app/products-sdk/agent-app-sdk/index.mdx

```diff
@@ -443,6 +443,75 @@ widget.setReportsFilters({ tag: ['chatbot'] });
 
 </CodeSample>
 
+## Settings widgets
+
+If you want to connect a Settings widget to the LiveChat Agent App, you should use the `createSettingsWidget` function. It returns a promise resolving to a Settings widget instance.
+
+<Code>
+<CodeSample path={'SETTINGS WIDGETS'}>
+
+```js
+import { createSettingsWidget } from "@livechat/agent-app-sdk";
+
+createSettingsWidget().then(widget => {
+  // build your logic around the widget
+});
+```
+
+</CodeSample>
+</Code>
+
+### Events
+
+#### `page_data`
+
+Emitted when widget in initialized. The handler will get the main window page data object as an argument.
+
+Listen for the page data changes:
+
+<Code>
+<CodeSample path={'PAGE_DATA'}>
+
+```js
+widget.on("page_data", pageData => {
+  // read the page data
+});
+```
+
+</CodeSample>
+</Code>
+
+### Methods
+
+- [Get page data](#get-page-data)
+- [redirect](#redirect)
+
+#### Get page data
+
+Gets the main window page data recorded most recently. Returns the `IPageData` object, which is identical to the one emitted by the `page_data` event or null (if no data were registered).
```

Comment:
```suggestion
Gets the most recently recorded main window page data and returns the `IPageData` object (identical to the one emitted by the `page_data` event). If no data was recorded, it returns null.
```

Reply:
also, is the `IPageData` correct?

Reply:
It's correct - https://github.com/livechat/products-sdk/blob/master/packages/agent-app-sdk/src/widgets/shared/page-data.ts#L3

Reply:
9304fdfe99adbde9b902385cc7cdae59b4d4bc4d - fixed copy